CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+# Contributing 
+This is a document outlining how to contribute to the Tūhura Tech wiki.
+
+There are two main ways of contributing:  
+- Suggesting - this involves creating a new [issue on Github](https://github.com/Tuhura-Tech/Wiki/issues) and informing us of a way to improve the wiki. Alturnatively, you can notify the team directly on our [Discord server](https://github.com/Tuhura-Tech/Wiki/issues)
+- Directly - to do this you need to [fork our repository](https://github.com/Tuhura-Tech/Wiki/fork) and editing it, then creating a pull request to merge with the main site
+
+## Issues and suggestions
+If you have a suggestion or an issue you have found with the wiki please go to our [Github issues](https://github.com/Tuhura-Tech/Wiki/issues) page and start a new issue
+
+### Creating an issue
+For a new issue on the [Github issues](https://github.com/Tuhura-Tech/Wiki/issues) issues page, press 'New issue'   
+From there decide which issue you would like to raise:
+- Docs Request - this is for if you have information that you would like to be documented on the Tūhura Tech wiki, or you have a request for something you would like us to look into and document
+- Report a security vulnerability - if you have seen a security vulnerability, let us know privately here
+
+### Documentation request
+
+#### Title
+Try and keep the title for what you want to be request short and simple.  
+A good title is no more than a few words, for example "Python Introduction" or "Database backend" are good titles.
+
+#### Short description
+This is to expand your request into a full sentence or two. Try and keep this simple without going into detail.  
+A good example of a short description is "Please write a guide on how to use markdown"
+
+#### Detailed/additional information 
+This is optional.  
+If you have information you would like to be cut down and added to the wiki add it here. 
+
+Or, if you have screenshots to help explain what you would like to be added paste them here.  
+The easiest way to add an image is to `right click > copy image` on most platforms like Google Chrome or Discord, and then using `ctrl + v` in the issue to paste it as a markdown link
+
+## Directly writing documentation
+It's awesome you want to help us document and create resources for people to learn from! To get started you will need to set a few things up
+
+### Setting up
+To contribute you first need to hold a copy of our wiki on your device to edit it. First off, create a [fork of the repository](https://github.com/Tuhura-Tech/Wiki/fork)
+
+The simplest way to clone [the repository](https://github.com/Tuhura-Tech/Wiki) is to use [Github Desktop](https://desktop.github.com).   
```

Comment:
They are going to need to use an IDE at some point otherwise it will be complicated so do we want to encourage cloning through VSCode or just point to their docs instead of having to maintain our own docs.

Reply:
I've been changing everything in notepad, what people use to edit the text for documentation doesn't matter. 

I will suggest that people use VSCode to edit the text though